types/yargs/package.json

```diff
@@ -1,5 +1,9 @@
 {
     "private": true,
+    "types": "index",
```

Comment:
...? Is there a reason this has to be added?

Reply:
I don't remember. Possibly due to the addition of `typesVersions`? But we can remove it and see if tests work fine without it.

Reply:
Yeah, it's required for tests to pass
```
$ npm test yargs

> definitely-typed@0.0.3 test
> dtslint types "yargs"

AssertionError [ERR_ASSERTION]: "types" in '/Users/forivall/code/repos/DefinitelyTyped/types/yargs/package.json' should be "index".
    at /Users/forivall/code/repos/DefinitelyTyped/node_modules/dtslint/bin/checks.js:33:20
    at Generator.next (<anonymous>)
    at fulfilled (/Users/forivall/code/repos/DefinitelyTyped/node_modules/dtslint/bin/checks.js:5:58)
```